k8s/postgresql/manifest/application.yaml.template

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
It looks like the application has changed the structure in the meantime. Can you rewrite this using the example of other solutions?

RabbitMQ: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/rabbitmq/manifest/application.yaml.template